tedana/metrics/collect.py

```diff
@@ -0,0 +1,140 @@
+"""
+Collect metrics.
+"""
+import logging
+
+import numpy as np
+import pandas as pd
+
+from .dependence import *
```

Comment:
```suggestion
from . import dependence
```

And then update the function calls accordingly. Not a huge fan of * imports within internal module, though that is a stylistic choice.